types/errors.go

```diff
@@ -1,12 +1,16 @@
 package types
 
```

Comment:
probably should refactor to this to a frauderr package for namespacing

Reply:
IMO out of scope of this pr.